pkg/apis/sources/v1alpha1/mongodbsource_lifecycle_test.go

```diff
@@ -104,6 +104,7 @@ func TestMongoDbGetCondition(t *testing.T) {
 			m := &MongoDbSourceStatus{}
```

Comment:
can you also add one new UT, saying something like mark sink, deployed and connection established?

Reply:
Will do!

Reply:
Oh forgot about that, will add tonite!

Reply:
ack, you can do it in a follow up

Reply:
Will do!